packages/upswyng-types/src/index.ts

```diff
@@ -1,5 +1,12 @@
 import { TTimezoneName as TTimezoneName_ } from "./TTimezoneName";
 
+export interface TAlgoliaHit {
+  objectID: string;
```

Comment:
consider adding a comment that for resources `objectID` = `resourceId`

Reply:
[updated](https://github.com/CodeForBoulder/upswyng/pull/278/commits/efb6ef5dbc5165c1ca97401eb5aee1b1d3c49358)!!

![ID is ID meme](https://i.imgflip.com/3q1q39.jpg)